### DISCUSSION TITLE
how can I use mc admin to turn off the LDAP login?

### DISCUSSION THREAD

**Discussion by andyzheung** [minio/minio#16681] [Q&A] [answered]
As above.

**Comment by harshavardhana** [comment, marked as answer]
- `mc admin config set alias/ identity_ldap enable=off` 

or 

- `mc admin config reset alias/ identity_ldap`

**Reply by andyzheung** [reply]
> * `mc admin config set alias/ identity_ldap enable=off`
> 
> or
> 
> * `mc admin config reset alias/ identity_ldap`

`mc admin config set alias/ identity_ldap enable=off`

mc: Unable to set 'identity_ldap enable=off' to server: sub-system 'identity_ldap' cannot have empty keys

I try to use this:
mc admin config reset alias/ identity_ldap

but when I login the console use rootuser and password, I found the ldap is still not disable??

LDAP server error: Unable to find user DN: User DN for

**Reply by harshavardhana** [reply]
Do this

```
mc admin config set alias/ $(mc admin config get play/ identity_ldap) enable=off
```

**Reply by andyzheung** [reply]
> mc admin config set alias/ $(mc admin config get play/ identity_ldap) enable=off

mc: <ERROR> Unable to set 'identity_ldap server_addr= srv_record_name= user_dn_search_base_dn= user_dn_search_filter= group_search_filter= group_search_base_dn= tls_skip_verify=off server_insecure=off server_starttls=off lookup_bind_dn= lookup_bind_password= enable=off' to server. LDAP Server Connection Error: Could not connect to LDAP server: LDAP Result Code 200 "Network Error": dial tcp: lookup srv_record_name=: no such host.

I think because I have do:
mc admin config reset alias/ identity_ldap
